cv-data-service-library/src/main/java/com/trihydro/library/helpers/JsonToJavaConverter.java

```diff
@@ -95,13 +64,12 @@ public OdeLogMetadata convertTimMetadataJsonToJava(String value) {
                     ((ObjectNode) metaDataNode).replace("receivedMessageDetails", receivedMessageDetailsNode);
                 }
             }
-            // System.out.println(metaDataNode);
+            log.trace("MetaDataNode: {}", metaDataNode);
```

Comment:
Note: This was a commented out System.out print and has been changed to a TRACE level log.